### ISSUE TITLE
[SpringMVC] Versioned, class-level RequestMappings do not render

### ISSUE THREAD

**Issue by who** [kongchen/swagger-maven-plugin#144]
If a class-level `@RequestMapping` is vX, where X is a digit, it is not rendered by swagger-maven-plugin.

For example, the class below will not exist in the output rendered by swagger-maven-plugin.

```
@RestController
@RequestMapping(value = "/v1")
@Api(value="TestingController")
public class TestingController {}
```

Looks like the bug is in this code: 

https://github.com/kongchen/swagger-maven-plugin/blob/c4031c968ffdc039e321a4940b75f97232ba2f42/src/main/java/com/github/kongchen/swagger/docgen/util/SprintUtils.java#L50-L55

**Comment by kongchen** [comment]
Please try 3.0.1-SNAPSHOT

**Comment by who** [comment]
Confirmed the fix on 3.0.1-SNAPSHOT.  Thanks
